docs/source/item-validation.rst

```diff
@@ -193,6 +221,38 @@ as a `dict`:
         OtherItem: '/path/to/otheritem_schema.json',
     }
 
+.. _SPIDERMON_VALIDATION_CERBERUS:
+
+SPIDERMON_VALIDATION_CERBERUS
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Default: ``None``
+
+A `list` containing the location of the item schema. Could be a local path, URL or the schema itself.
+
+.. code-block:: python
+
+    # settings.py
+
+    SPIDERMON_VALIDATION_CERBERUS = [
+        '/path/to/schema.json',
+        'http://example.com/mycerberusschema',
+        {"Field": {"type": "number", "required":True}}
+    ]
+
+If you are working on a spider that produces multiple items types, you can define paths to schema for each item as `dict` as shown below:
+
+.. code-block:: python
+
+    # settings.py
+
+    from quotes.items import DummyItem, OtherItem
+
+    SPIDERMON_VALIDATION_CERBERUS = {
+        DummyItem: '/path/to/dummyitem_schema.json',
+        OtherItem: '/path/to/otheritem_schema.json',
+    }
+
```

Comment:
Here the documentation says that per-item-type schemas are useful when your spider produces multiple item types.

That makes it seem like having a list with multiple schemas (first syntax above) is useful even if you have a single item type.

If that is the case, then I would expect the dictionary syntax to support multiple schemas per item type, instead of a single one as it currently does.

So, I think we should do one of these:
- Rephrase the documentation, so that the dictionary syntax is recommended not for *spiders that produce multiple items types*, but for *enforcing a specific schema based on item type*.
- Allow a list of schemas as well when using a dictionary syntax, so that multiple schemas can be assigned to any given issue type.

Reply:
On hindsight, @rennerocha and I had a bit of discussion regarding different schemas for different item types. I felt that there is no real applicable use for having the scenario I mentioned above. I might be wrong here, but I feel while having a list of schemas as well for dictionary syntax might be the right way forward. I am not usre it would be used as much. Let me know what you think. I am happy with changing the docs, so I can go ahead with one.

Reply:
@Gallaecio and @vipulgupta2048 
We are already allowed to have a schema list in the dictionary as we can see [in this function](https://github.com/scrapinghub/spidermon/blob/master/spidermon/contrib/scrapy/pipelines.py#L47), [here](https://github.com/scrapinghub/spidermon/blob/master/spidermon/contrib/scrapy/pipelines.py#L49) and [here](https://github.com/scrapinghub/spidermon/blob/master/spidermon/contrib/scrapy/pipelines.py#L52), but I believe that this is very unusual, but it could be described in the documentation.

Reply:
Roger that, rephrasing the docs.